### DISCUSSION TITLE
Help needed on new server: image doesn't show up

### DISCUSSION THREAD

**Discussion by moniesoke** [mastodon/mastodon#20287] [General]
Hi All,

I setup a server on a Oracle Cloud instance that is running Ubuntu 22. I can post/upload images and videos. I can see the images and videos in my server. But they do not show up. I cannot play the video. it has a play button, but when I click it, it just kept loading and time out. The website is videochina.net

I am wondering whether it is a firewall issue or database issue?

**Comment by mitexleo** [comment]
Did you precompiled assets while installing?

**Reply by moniesoke** [reply]
Hi, Thanks for the hint. I don't know what that means. I followed the instructions on mastodon website about installing from source (https://docs.joinmastodon.org/admin/install/) and run every command it asked me to do. The result is not very good. 

The web doesn't work but the mobile app works and The images are wired.

Anyway, I ended up using the preinstalled image from digitalocean. Now everything is working alright, sort of.
